CMakeLists.txt

```
@@ -71,15 +71,15 @@ target_compile_definitions(vulkanCapsViewer PRIVATE ${VULKANCAPSVIEWER_DEFINITIO
 
 if(NOT APPLE)
     if(WIN32)
-        target_link_libraries(vulkanCapsViewer Qt5::Core Qt5::Gui Qt5::Widgets Qt5::Network "${VULKAN_LOADER_INSTALL_DIR}/lib/vulkan-1.lib")
+        target_link_libraries(vulkanCapsViewer Qt6::Core Qt6::Gui Qt6::Widgets Qt6::Network "${VULKAN_LOADER_INSTALL_DIR}/lib/vulkan-1.lib")
     else()
         # Linux...
 	if(X11)
-	    target_link_libraries(vulkanCapsViewer Qt5::Core Qt5::Gui Qt5::Widgets Qt5::Network Qt5::x11extras "${VULKAN_LOADER_INSTALL_DIR}/lib/vulkan")
+	    target_link_libraries(vulkanCapsViewer Qt6::Core Qt6::Gui Qt6::Widgets Qt6::Network Qt6::x11extras "${VULKAN_LOADER_INSTALL_DIR}/lib/vulkan")
         elseif(WAYLAND)
-            target_link_libraries(vulkanCapsViewer Qt5::Core Qt5::Gui Qt5::Widgets Qt5::Network Qt5::waylandclient "${VULKAN_LOADER_INSTALL_DIR}/lib/vulkan")
+            target_link_libraries(vulkanCapsViewer Qt6::Core Qt5::Gui Qt6::Widgets Qt6::Network Qt6::waylandclient "${VULKAN_LOADER_INSTALL_DIR}/lib/vulkan")

```

You left a `Qt5::` in there...

```suggestion
            target_link_libraries(vulkanCapsViewer Qt6::Core Qt6::Gui Qt6::Widgets Qt6::Network Qt6::waylandclient "${VULKAN_LOADER_INSTALL_DIR}/lib/vulkan")

```

---

So I did. Thanks again!